cpp/src/arrow/util/bitmap_ops.cc

```diff
@@ -248,13 +250,15 @@ void ReverseBitmap(const uint8_t* data, int64_t offset, int64_t length, uint8_t*
 }
 
 Result<std::shared_ptr<Buffer>> CopyBitmap(MemoryPool* pool, const uint8_t* data,
-                                           int64_t offset, int64_t length) {
-  return TransferBitmap<TransferMode::Copy>(pool, data, offset, length);
+                                           int64_t offset, int64_t length,
+                                           int64_t out_offset) {
+  return TransferBitmap<TransferMode::Copy>(pool, data, offset, length, out_offset);
 }
 
 Result<std::shared_ptr<Buffer>> InvertBitmap(MemoryPool* pool, const uint8_t* data,
                                              int64_t offset, int64_t length) {
```

Comment:
I am not adding `out_offset` to `InvertBitmap` because I am unsure how useful that could really be.